modules/gitops-deploy-resources-different-namespaces.adoc

```diff
@@ -14,8 +14,8 @@ To allow Argo CD to manage resources in other namespaces apart from where it is
 +
 [source,terminal]
 ----
-$ oc label namespace <namespace> \
-argocd.argoproj.io/managed-by=<namespace> <1>
+$ oc label namespace <target-namespace> \
+argocd.argoproj.io/managed-by=<argocd-namespace> <1>
 ----
 <1> The namespace where Argo CD is installed. 
```

Comment:
* Configure the target namespace by running the following command:
+
[source,terminal]
----
$ oc label namespace <target_namespace> \
argocd.argoproj.io/managed-by=<argocd_namespace> 
----
+
where:

`<target_namespace>`:: Specifies the name of the namespace you want Argo CD to manage.
`<argocd_namespace>`:: Specifies the name of the namespace where Argo CD is installed.